src/overlays/actors/ovl_Bg_F40_Switch/z_bg_f40_switch.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "z_bg_f40_switch.h"
+#include "z64actor.h"
```

Comment:
This should not be necessary: this header is included in global.h, which is included in the actor header.